### ISSUE TITLE
Unable to to resolve AccessibilityInfo from default project

### ISSUE THREAD

**Issue by nsmith1024** [facebook/react-native#20865]
- [ ] Review the documentation: https://facebook.github.io/react-native
  - [ ] Search for existing issues: https://github.com/facebook/react-native/issues
  - [ ] Use the latest React Native release: https://github.com/facebook/react-native/releases

## Environment
Environment:
  OS: Windows 8.1
  Node: 8.11.4
  Yarn: 1.9.4
  npm: 5.6.0
  Watchman: Not Found
  Xcode: N/A
  Android Studio: Not Found

Packages: (wanted => installed)
  react: 16.3.1 => 16.3.1
  react-native: 0.55.4 => 0.55.4

## Description
Steps
1. install react-native-cli on windows
2. react-native init p1 --version 0.55.4
3. cd p1
4. react-native run-android

I have android phone plugged into USB port

I get red screen with error cant resolve AccessibilityInfo

## Reproducible Demo
I did not modify any code, i used the default project created using "react-native init p1  --version 0.55.4"

using "react-native init p1"  has the same error, even without changing anything

**Comment by abdulrahman-khankan** [comment]
Happens here with latest release
```
    "react": "16.4.1",
    "react-native": "^0.56.0"
```

**Comment by donjo9** [comment]
#19953 and #20353 already mentioned this :) 

Try out `react-native init p1 --version 0.57.0-rc.3`
 if you run into #20791 while trying, follow the instructions in the bottom of that post
